src/components/RefinementList.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div :class="suit('')" v-if="state">
+    <slot
+      :items="state.items"
+      :refine="state.refine"
+      :searchable="searchable"
+      :search-for-items="state.searchForItems"
+      :toggle-show-more="toggleShowMore"
+      :is-showing-more="state.isShowingMore"
+      :createURL="state.createURL"
+      :is-from-search="state.isFromSearch"
+      :can-refine="state.canRefine"
+      :transform-Item-To-Hit="transformItemToHit"
+    >
+      <div :class="suit('searchBox')" v-if="searchable">
+        <ais-search-input v-model="searchForFacetValues"></ais-search-input>
+      </div>
+      <ul :class="suit('list')">
+        <li
+          :class="[suit('item'), {[suit('item', 'selected')]: item.isRefined}]"
+          v-for="item in state.items"
+          :key="item.value"
+        >
+          <slot name="item" :item="item" :refine="state.refine">
+            <label :class="suit('label')">
+              <input
+                :class="suit('checkbox')"
+                type="checkbox"
+                :value="item.value"
+                :checked="item.isRefined"
+                @change="state.refine(item.value)"
+              />
+              <span v-if="searchable" :class="suit('labelText')">
+                <ais-highlight attribute="item" :hit="transformItemToHit(item)"/>
+              </span>
+              <span v-else :class="suit('labelText')">{{item.label}}</span>
+              <span :class="suit('count')">{{item.count}}</span>
+            </label>
+          </slot>
+        </li>
+      </ul>
+      <button
+        :class="[suit('showMore'), { [suit('showMore', 'disabled')]: !state.canToggleShowMore }]"
+        @click="toggleShowMore"
+        v-if="showMore"
+        :disabled="!state.canToggleShowMore"
+      >
+        <slot name="showMoreTitle" :is-showing-more="state.isShowingMore">
```

Comment:
We use `label` rather than `title` on the other widgets. We can review this differences in one pass in another PR though.